### PR TITLE
feat(storage): simply support trigger manual compaction and risectl

### PR DESCRIPTION
**Pull request by Li0k**
## What's changed and what's your intention?

simply support trigger manual compaction

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:

- simply support trigger manual compaction in hummock_manager
- add trigger manual compaction rpc
- support  trigger manual compaction for risectl

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
## Refer to a related PR or issue link (optional)
- https://github.com/singularity-data/risingwave/issues/3097

### CODE REVIEW

**Review by skyzh — APPROVED** [review]

**Comment by Li0k** [comment]
I found a bug that:  after pick_compaction, compaction_status will hold the pending_task in memory. so if send_task rpc fail,

File: src/meta/src/hummock/hummock_manager_tests.rs [hunk 1]
```diff
@@ -915,3 +915,102 @@ async fn test_mark_orphan_ssts() {
...
+        .await
+        .unwrap()
+        .unwrap();
+
+    // safe epoch should be INVALID before success compaction
+    assert_eq!(INVALID_EPOCH, hummock_version1.safe_epoch);
+
+    {
+        // to check compactor send task fail
+        drop(_receiver);
```
Comment (skyzh) [inline src/meta/src/hummock/hummock_manager_tests.rs:981]: drop(receiver);

if a variable is used, remove the `_` prefix

File: src/meta/src/hummock/hummock_manager.rs [hunk 2]
```diff
@@ -1330,10 +1337,105 @@ where
...
+                    .unwrap();
+                compact_status.cancel_compaction_tasks_if(|pending_task_id| {
+                    pending_task_id == compact_task.task_id
+                });
+
+                return Err(error);
+            }
+        }
+
+        tracing::trace!(
```
Comment (skyzh) [inline src/meta/src/hummock/hummock_manager.rs:1420]: maybe change the log level to info, otherwise in release mode this won't be printed

**Review by skyzh — APPROVED** [review]
Rest LGTM

File: src/meta/src/hummock/hummock_manager.rs (hunk 2 above)
Comment (Li0k) [reply src/meta/src/hummock/hummock_manager.rs]: fix it

**Pull request merged**